src/parser/create_statement.cpp

```diff
@@ -0,0 +1,129 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+// table_ref.cpp
+//
+// Identification: src/parser/create_statement.cpp
+//
+// Copyright (c) 2015-17, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#include "parser/create_statement.h"
+
+namespace peloton {
+namespace parser {
+
+const std::string CreateStatement::GetInfo(int num_indent) const {
+  std::ostringstream os;
+  os << StringUtil::Indent(num_indent) << "CreateStatement\n";
+  os << StringUtil::Indent(num_indent + 1);
+
+  switch (type) {
+    case CreateStatement::CreateType::kTable: {
+      os << "Create type: Table"
+         << "\n";
```

Comment:
Also, move all of these newlines at the end of a `case` statement to be outside the `switch`

Reply:
Done